src/components/atoms/form/ExecutorProperty.tsx

```diff
@@ -0,0 +1,85 @@
+import { orb, reusable } from '@circleci/circleci-config-sdk';
```

Comment:
Is this a new feature? not part of a refactor.

Reply:
This was already supported, just broke it out and just mixed it in with this refactor.